README.md

```diff
@@ -1,6 +1,26 @@
-# jshint loader for webpack
+<div align="center">
+<a href="https://github.com/webpack/webpack">
+    <img width="200" height="200"
+      src="https://webpack.js.org/assets/icon-square-big.svg">
+  </a>
+  </div>
+<h2 align="center">jshint loader for webpack</h2>
 
-## Usage
+<h2 align="center">Install</h2>
+
+### npm
+
+```bash
+npm install --save-dev jshint-loader jshint
+```
+
+### yarn
+
+```bash
+yarn add --save-dev jshint-loader jshint
```

Comment:
I believe it should be `yarn add --dev jshint-loader jshint`, [docs](https://yarnpkg.com/en/docs/cli/add#toc-yarn-add-dev-d).